### PR TITLE
finished css-positioning

### PR DESCRIPTION
**Pull request by michael5522**
Screen Shot 2022-02-10 at 6 57 06 PM
Screen Shot 2022-02-10 at 7 09 07 PM
Screen Shot 2022-02-10 at 7 12 19 PM

### CODE REVIEW

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Make sure the overlay is covering the image good sir!

**Comment by michael5522** [comment]
Screen Shot 2022-02-10 at 7 35 00 PM

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  top: 160px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
+
```
Comment (BrettAlbright) [inline css-positioning/absolute.css:40]: you should have `position:absolute` here

**Review by BrettAlbright — APPROVED** [review]
Make sure to update the position property!

**Pull request merged**